test/inductor/test_smoke.py

```diff
@@ -1,10 +1,11 @@
 # Owner(s): ["module: inductor"]
 import logging
+import unittest
 
 import torch
 import torch._dynamo as torchdynamo
 import torch._inductor.config as torchinductor_config
-from torch.testing._internal.common_utils import IS_LINUX, TestCase
+from torch.testing._internal.common_utils import IS_JETSON, IS_LINUX, TestCase
```

Comment:
You don't need is jetson here I guess, but would be nice to add
```suggestion
from torch.testing._internal.common_utils import IS_LINUX, TestCase
from torch.testing._internal.inductor_utils import HAS_CUDA
```